# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hyprland/hotfix-no-liftoff.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- hyprland-source/CMakeLists.txt.vanilla	2024-09-30 22:11:15.240000000 +0100
+++ hyprland-source/CMakeLists.txt	2024-09-30 22:11:21.396000000 +0100
@@ -114,7 +114,6 @@
   hwdata
   libseat
   libdisplay-info
-  libliftoff
   libudev
   gbm
   hyprlang>=0.3.2
